Sources/HipMobileUI/Resources/Strings.resx

```diff
@@ -816,17 +816,23 @@ Verantwortlich für den Betrieb: Dr. Simon Oberthür</value>
   </data>
   <data name="AchievementsScreenView_Score" xml:space="preserve">
     <value>Dein Punktestand: </value>
-  </data>	  
+  </data>
   <data name="AchievementsScreenView_PleaseLogin" xml:space="preserve">
     <value>Um Achievements zu nutzen, musst du dich zunächst einloggen.</value>
-  </data>	
+  </data>
   <data name="MainPageViewModel_Achievements" xml:space="preserve">
     <value>Achievements</value>
-  </data>	
+  </data>
   <data name="ExhibitNearby_VisitRequest" xml:space="preserve">
     <value>Besuche eine nahe Sehenswürdigkeit!</value>
   </data>
-  <data name="AchievementsDetailsExhibitView_VisitedMOfNExhibits">
+  <data name="AchievementsDetailsExhibitView_VisitedMOfNExhibits" xml:space="preserve">
     <value>Du hast {0} von {1} Sehenswürdigkeiten besucht</value>
   </data>
+  <data name="SkipExhibit_Message" xml:space="preserve">
+    <value>Alle Sehenswürdigkeiten wurden bereits besucht. Es kann keine weitere Alle Sehenswürdigkeiten wurden bereits besucht. Es kann keine weitere übersprungen werden.</value>
```

Comment:
This doesn't look right. Parts of the message are duplicates. The content of `value ` should be as follows:

"Alle Sehenswürdigkeiten wurden bereits besucht. Es kann keine weitere übersprungen werden."